plugins/org.yakindu.sct.ui.editor/src/org/yakindu/sct/ui/editor/editor/themes/ColorHelper.java

```diff
@@ -0,0 +1,171 @@
+package org.yakindu.sct.ui.editor.editor.themes;
+
+import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.graphics.RGB;
+
+public class ColorHelper {
+	public static double getShadePercentage(Color color) {
+		HSL hsl = HSL.fromRGB(color.getRGB());
```

Comment:
Is this HSL color space really required? Something that can not be done with RGB + libs?

Reply:
Yeah, HSL allows to transform colors programmatically which is not really possible in RGB, and there doesn't seem to be a good library offering HSL conversion.